### ISSUE TITLE
Playback gets choppy with Pixel Bar and / or Timeline

### ISSUE THREAD

**Issue by SteffenDuenner** [ggarra13/mrv2#117]
I'm currently having to deal with 6k footage and noticed that playback gets extremely choppy as soon as either the Pixel Bar and / or the Timeline is visible. As soon as they're both hidden playback is fine and stable.

I noticed the performance drop with the Pixel Bar some time ago (and IIRC reported it) but I can live without a pixel bar. I can also totally understand that the performance suffers if it has to pick a pixel value from a hight resolution image every single frame.
But now I also noticed that the Timeline does the same thing, which technically makes no sense to me (as a mere enduser ;) )

**Comment by ggarra13** [comment]
Some questions:

1) Platform you are on (Linux right?).
2) Can you save out 30 images in a .zip file and upload them somewhere?
3) Do you have the Edit mode on (displaying the thumbnails)?

**Comment by SteffenDuenner** [comment]
Thanks for the quick reaction and sorry for the insufficient info from my side:

1. Platform Linux indeed (Linux Mint 21.2)
2. I zipped 30 frames and uploaded them here:
https://1drv.ms/u/s!AlOpRG0iPvzdhKtUQrRKDgB4KjXYwA?e=milXgp
This sequence is based on footage coming from Blackmagic. I converted them to DWAB EXRs.
3. Edit mode is off
![image](https://github.com/ggarra13/mrv2/assets/33224667/605bada4-4111-410a-8992-6e523f6fb4ca)
